docs/content/troubleshooting/log_status_collect.md

```diff
@@ -1,41 +1,101 @@
 
-## Detecting errors
+**Important:** Be sure that the logs cover any relevant time frames for the specific issues that you are trying to debug when gathering logs from the storage system.
 
-To help pinpoint potential causes for stateful pod failure:
+**Note:** All commands here are listed with the collection command with the `logs` folder name example. Change the folder name according as needed.
 
-1.  Verify that all CSI pods are running.
+### General status and log collection for IBM Support
+Be sure to run the following steps and copy the output to an external file, when engaging IBM Support and sending log collections.
 
-    ```
-    kubectl get pods -n <namespace> -l csi
-    ```
+1. Check the node status.
+    
+    `kubectl get nodes`
+2. Check the CSI driver component status.
 
-2.  If a pod is not in a _Running_ state, run the following command:
+    `kubectl get -all-namespaces pod -o wide | grep ibm-block-csi`
+3. Check if the PersistentVolumeClaims (PVCs) are _Bound_.
 
-    ```
-    kubectl describe -n <namespace> pod/<pod-name>
-    ```
+    `kubectl get pvc`
 
-    View the logs.
+    **Note:** If the PVCs are not in the _Bound_ state collect the events of all unbound PVCs. (See [Log collection for unbound PVCs](#log-collection-for-unbound-pvcs).)
+
+### Log collection for all CSI driver node pods and their containers
+
+To collect logs for all CSI driver node pods, use the following commands:
+
+    nodepods=`kubectl get pods -l product=ibm-block-csi-driver -l app.kubernetes.io/component=csi-node --output=jsonpath={.items..metadata.name}`
+    
+    for pod in $nodepods;do for container in `kubectl get -n <namespace> pod $pod -o jsonpath='{.spec.containers[*].name}'`;do kubectl logs -n <namespace> $pod -c $container > logs/${pod}_${container}.log;done;done
+
+
+### Log collection for all CSI controller containers
+
+To collect logs for all pods and containers, use the following commands:
+    
+    for container in `kubectl get -n <namespace> pod ibm-block-csi-controller-0 -o jsonpath='{.spec.containers[*].name}'`;do kubectl logs -n <namespace> ibm-block-csi-controller-0 -c $container > logs/ibm-block-csi-controller-0_${container}.log;done
+
+
+### Log collection for CSI operator logs
+To collect CSI operator logs, use the following commands:
+
+    operatorpod=`kubectl get pods --all-namespaces |grep ibm-block-csi-operator|awk '{print $2}'`
+    kubectl logs $operatorpod -n <namespace> > logs/operator
+
+
+### Collecting details of all CSI objects and components
+`kubectl describe all -l product=ibm-block-csi-driver -n <namespace> > logs/describe_ibm-block-csi-driver`
+
+
+### Status collection for CSI pods, daemonset, and statefulset
+`kubectl get all -n <namespace> -l product=ibm-block-csi-driver > logs/get_all_ibm-block-csi-driver`
+
+
+
+### Log collection for the CSI driver controller
```

Comment:
Why do we need this? We have the same and more in line 63.

Reply:
@roysahar  Does 
`for container in `kubectl get -n <namespace> pod ibm-block-csi-controller-0 -o jsonpath='{.spec.containers[*].name}'`;do kubectl logs -n <namespace> ibm-block-csi-controller-0 -c $container > logs/ibm-block-csi-controller-0_${container}.log;done`

do the same thing as 
`kubectl logs -f -n <namespace> ibm-block-csi-controller-0 -c ibm-block-csi-controller > logs/ibm-block-csi-controller``

Reply:
the `for` loop makes the output per file, which is important when dealing with support

Reply:
As per my discussion with @oriyarde  I am going to indent three of the "all" commands under the commands for Support.

The other "smaller" similar commands are for the user to use so that they have more directed log output to troubleshoot specific needs.

Reply:
But the "small" command will show only the one of the containers (`ibm-block-csi-controller`) in the `ibm-block-csi-controller-0` POD. This can lead the customer not to see the reason for the failure/s he is looking.

Reply:
I think that the information is there and they can always run the other commands but this way they're not overwhelmed with information.  the support commands that we're giving are almost a "courtesy" to support so that they get the fullest picture possible when contacted.

@oriyarde  what do you think? leave or remove?

Reply:
OK